File: patterns/c-patterns/pattern18.c
// Pattern 18: ...

// 55555
// *4444
// **333
// ***22
// ****1

#include <stdio.h>
int main()
{
    int i, j;
    int n = 5;
    for (i = n; i >= 1; i--)
    {
        for (j = n; j >= 1; j--)
        {
            if (j > i)
                printf("*");
            else
                printf("%d", i);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern18.py
# Pattern 18: ...

# 55555
# *4444
# **333
# ***22
# ****1

n = 5

for i in range(n, 0, -1):
    for j in range(n, 0, -1):
        if j > i:
            print("*", end="")
        else:
            print(i, end="")
    print()
